src/utils/analytics/index.ts

```diff
@@ -0,0 +1,58 @@
+import noop from 'utils/noop'
+import {identify} from './identify'
+import {
+  activityInternalLinkClick,
+  activityExternalLinkClick,
+  activityCtaClick,
+  engagementSearchedWithQuery,
+  engagementWatchedTalk,
+  engagementStartCourse,
+  engagementCourseProgress,
+  engagementCompletedCourse,
+  engagementCompletedLesson,
+  engagementClickedWatchedLessonAgain,
+  engagementListenPodcast,
+  engagementReadArticle,
+} from './events'
+import {
+  purchaseSubscriptionUpgraded,
+  purchaseSubscriptionDowngraded,
+  purchaseSetSubscriptionStatus,
+  purchaseSubscriptionCanceled,
+  purchaseSubscriptionCreated,
+} from './server-side-events'
+
+const serverSideEvents = {
+  purchaseSubscriptionUpgraded,
+  purchaseSubscriptionDowngraded,
+  purchaseSetSubscriptionStatus,
+  purchaseSubscriptionCanceled,
+  purchaseSubscriptionCreated,
+}
+
+const events = {
+  activityInternalLinkClick,
+  activityExternalLinkClick,
+  activityCtaClick,
+  engagementSearchedWithQuery,
+  engagementWatchedTalk,
+  engagementStartCourse,
+  engagementCourseProgress,
+  engagementCompletedCourse,
+  engagementCompletedLesson,
+  engagementClickedWatchedLessonAgain,
+  engagementListenPodcast,
+  engagementReadArticle,
+}
+
+export const track = (event: string, paramsOrCallback?: any, callback?: any) =>
+  noop
```

Comment:
the old track calls throughout the sight are now a `noop` so we should be starting fresh after this PR